app/src/main/java/ch/epfl/sdp/blindly/chat/Message.kt

```diff
@@ -1,5 +1,8 @@
 package ch.epfl.sdp.blindly.chat
 
+/**
```

Comment:
I also added a comment in my PR  #182 we will have to take care of that when mergin #182. Especially since this class changes a bit.

Reply:
Okay ! I'll take care of it.